src/test/java/org/openmrs/contrib/qaframework/page/ConditionsPage.java

```diff
@@ -25,7 +25,8 @@
 	private static final By TAB_ACTIVE = By.cssSelector("a[href='#ACTIVE']");
 	private static final By TAB_INACTIVE = By.cssSelector("a[href='#INACTIVE']");
 	private static final By FIRST_CONDITION_NAME = By.xpath("//table/tbody[2]/tr[1]/td[1]");
-	private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
+	private static final By EDIT_ACTIVE_CONDITION = By.cssSelector("#ACTIVE tbody:nth-child(3) i.icon-pencil.edit-action.ng-scope");
```

Comment:
is the element retrivable if you get rid of the nth-value?
```suggestion
	private static final By EDIT_ACTIVE_CONDITION = By.cssSelector("#ACTIVE i.icon-pencil.edit-action.ng-scope");
```

Reply:
> is the element retrievable if we get rid of the `nth-value`?

it shows 2 elements. that is as of my local instance. still waiting for the server to get back up and confirm

Reply:
`#ACTIVE tbody.ng-scope i.icon-pencil.edit-action.ng-scope` is the best option.